### PR TITLE
aws-lightsail: add page

### PR DESCRIPTION
**Pull request by ShivamJoker**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): 2.8.2**

### CODE REVIEW

**Comment by ShivamJoker** [comment]
Mentioned in #5653

File: pages/common/aws-lightsail.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# aws lightsail
+
+> CLI for Lightsail.
+> You can manage your Lightsail resources using the cli.
+> More information: <https://awscli.amazonaws.com/v2/documentation/api/latest/reference/lightsail/index.html>.
+
+- List all Amazon Lightsail virtual private servers, or instances:
```
Comment (navarroaxel) [inline pages/common/aws-lightsail.md:7]: all these examples are for Lightsail
```suggestion
- List all virtual private servers, or instances:
```

File: pages/common/aws-lightsail.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+`aws lightsail get-instances`
+
+- List all Lightsail bundles (instance plans):
```
Comment (navarroaxel) [inline pages/common/aws-lightsail.md:11]:
```suggestion
- List all bundles (instance plans):
```

File: pages/common/aws-lightsail.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+- List all available instance images, or blueprints:
+
+`aws lightsail list-blueprints`
+
+- Create an instance:
+
+`aws lightsail create-instances --instance-names {{name}} --availability-zone {{ap-southeast-1a}} --bundle-id {{nano_2_0}} --blueprint-id {{nodejs}}`
+
+- Get instance state:
```
Comment (navarroaxel) [inline pages/common/aws-lightsail.md:23]: `get` is most for programming
```suggestion
- Print the state of a specific instance:
```

File: pages/common/aws-lightsail.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+`aws lightsail get-instance-state --instance-name {{name}}`
+
+- Stop instance:
```
Comment (navarroaxel) [inline pages/common/aws-lightsail.md:27]:
```suggestion
- Stop a specific instance:
```

File: pages/common/aws-lightsail.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+`aws lightsail stop-instance --instance-name {{name}}`
+
+- Delete instance:
```
Comment (navarroaxel) [inline pages/common/aws-lightsail.md:31]:
```suggestion
- Delete a specific instance:
```

File: pages/common/aws-lightsail.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+`aws lightsail list-bundles`
+
+- List all available instance images, or blueprints:
+
+`aws lightsail list-blueprints`
+
+- Create an instance:
+
+`aws lightsail create-instances --instance-names {{name}} --availability-zone {{ap-southeast-1a}} --bundle-id {{nano_2_0}} --blueprint-id {{nodejs}}`
```
Comment (navarroaxel) [inline pages/common/aws-lightsail.md:21]: :thinking: we use `region` in several AWS pages so. Also the blueprint ID comes from a similar API
```suggestion
`aws lightsail create-instances --instance-names {{name}} --availability-zone {{region}} --bundle-id {{nano_2_0}} --blueprint-id {{blueprint_id}}`
```

**Review by navarroaxel — APPROVED** [review]

**Comment by navarroaxel** [comment]
@ShivamJoker you don't need to keep the PR in sync `main` because the pages are independent.

**Comment by ShivamJoker** [comment]
> @ShivamJoker you don't need to keep the PR in sync `main` because the pages are independent.

Got it. Thanks!

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

> @navarroaxel I can't merge it as CLA workflow is waiting

**Comment by navarroaxel** [comment]
@kbdharun I think it's a bug with the CLA check, because the author already merged #8825, and the CLA passed when this PR were created, otherwise we had the bot's comment. I'll force the merge.

**Pull request merged**